pkg/virt-config/deprecation/feature-gates.go

```diff
@@ -76,6 +78,7 @@ func init() {
 
 	RegisterFeatureGate(FeatureGate{Name: PasstGate, State: Discontinued, Message: PasstDiscontinueMessage, VmiSpecUsed: passtApiUsed})
 	RegisterFeatureGate(FeatureGate{Name: MacvtapGate, State: Discontinued, Message: MacvtapDiscontinueMessage, VmiSpecUsed: macvtapApiUsed})
+	RegisterFeatureGate(FeatureGate{Name: DockerSELinuxMCSWorkaround, State: Deprecated, Message: fmt.Sprintf("DockerSELinuxMCSWorkaround has been discontinued since v1.4.")})
```

Comment:
I noticed it while rebasing over this:

It is marked as "Deprecated" but commented in the message as "discontinued".
There is a difference and a special state for both.

Reply:
Nice catch!

Reply:
I will create a followup to fix this

Reply:
https://github.com/kubevirt/kubevirt/pull/13042